src/main/webapp/jsp/adminEmail.jsp

```diff
@@ -6,6 +6,7 @@
 
 <c:set var="jsIncludes">
     <script type="text/javascript" src="/js/adminEmail.js"></script>
+    <script type="text/javascript" src="/js/richTextEditor.js"></script>
     <script type="text/javascript" src="/tinymce/js/tinymce/tinymce.min.js"></script>
```

Comment:
Let's do an improvement here: reorder the imports to tinymce -> richtexteditor -> adminemail to show the logical flow of which import depends on which.
Also, since I made a change on this line in #5719, you have to de-conflict/rebase as well.